R/dribble.R

```diff
@@ -0,0 +1,49 @@
+dribble <- function() {
+  structure(
+    tibble::tibble(
+      name = character(),
+      id = character(),
+      drive_file = list()
+    ),
+    class = c("dribble", "tbl_df", "tbl", "data.frame")
+  )
+}
```

Comment:
This is sort of handy when you want to return a dribble with zero rows. It is also an obvious place to put (internal) documentation about what the columns are.